docs/iot/iot-partners/thingspark.md

```diff
@@ -0,0 +1,27 @@
+# Tizen IoT with ThingsPark Cloud
+
+The ThingsPark service is a cloud-based IoT solution that can be connected with various things. 
+
+Tizen developers can easily develop an application to send IoT data to the cloud in real time.
```

Comment:
Please fix article and change "real time" to "real-time".
Tizen developers can easily develop applications that send IoT data to the cloud in real-time.